instagram/src/App.js

```diff
@@ -1,28 +1,32 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import dummyData from './dummy-data'; //import data contained in dummy-data.js
+
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer';
 
 class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      posts: []
+    }
+  }
```

Comment:
it's not a necessity since it will work regardless, but something interesting I shared with the group last week is that since there are no props being passed down to App (makes sense, since it is the first rendered component) you can actually do without the constructor and super here and declare state just with `state={}` instead of `this.state`. It's good to be in the habit of writing this part out since any child class component will need the constructor to take in props, but it's still useful to know